cfme/services/dashboard/ssui.py

```diff
@@ -81,33 +117,39 @@ def retiring_soon(self):
     view.flash.assert_no_error()
     view = self.create_view(MyServicesView)
     assert view.is_displayed
-    return retiring_services
+    view.paginator.set_items_per_page("100 items")
+    rows = view.paginator.items_amount
+    assert rows == retiring_services
```

Comment:
Same as with ``total_services``, consider if it would not be better to just return number of services.